emacs/merlin-iedit.el

```diff
@@ -36,18 +40,19 @@
         (iedit-hide-unmatched-lines iedit-occurrence-context-lines))))
   (length occurrences))
 
+;;;###autoload
 (defun merlin-iedit-occurrences ()
-  "Edit occurrences of identifier under cursor using `iedit'"
+  "Edit occurrences of identifier under cursor using `iedit'."
   (interactive)
   (if (bound-and-true-p iedit-mode) (iedit-mode -1)
-    (let ((r (merlin/call "occurrences"
-                          "-identifier-at" (merlin/unmake-point (point)))))
+    (let ((r (merlin-call "occurrences"
+                          "-identifier-at" (merlin-unmake-point (point)))))
       (when r
         (if (listp r)
             (flet ((iedit-printable (a)
```

Comment:
How about replacing the `flet` here with a `cl-letf` that would remove a usage of a deprecated function?
```suggestion
            (cl-letf ((iedit-printable (a)
```

Reply:
Thanks, that's not quite the right formulation because it's not a drop-in equivalent, but I've made a subsequent change in this spirit. :+1: